src/vision/status_quo/barbara_wishes_for_easy_runtime_switch.md

```diff
@@ -0,0 +1,76 @@
+# 😱 Status quo stories: Barbara wishes for an easy runtime switch
+
+## 🚧 Warning: Draft status 🚧
+
+This is a draft "status quo" story submitted as part of the brainstorming period. It is derived from
+real-life experiences of actual Rust users and is meant to reflect some of the challenges that Async
+Rust programmers face today.
+
+## The story
+
+[Barbara] has been working on an async codebase for the past 5 years. It is extremely mature and
+quite large (in the millions of lines of code). They've been relying on tokio as their async runtime
+and the codebase makes heavy use of its rich API. It has served them well over the years and they're
+very happy with it.
+
+Meanwhile, another team in Barbara's company have developed another large project, where they chosen
```

Comment:
I'm worried this will shift the story too much towards runtime agnostic libraries. If it's easy to write runtime agnostic libraries than this is a non issue. However, the original intent of the story was to motivate the need for runtime agnostic application code. These are related but IMO distinct issues.

Reply:
Good point but even if it was possible and easy to write runtime-agnostic libraries, does it mean everyone would?

Reply:
It seems like it's good enough for her to just wonder "hmm, maybe I should try out async-std for my application"...

Reply:
PTALA.